adage.snakefile

```diff
@@ -0,0 +1,36 @@
+# This snakefile relies on the outputs from add_new_to_compendia.snakefile,
+# specifically the file outputs/filt_norm_compendia/*compendium_p2_filtered_num_reads.csv
+# It uses this file as well as sets of samples defined in the R script to perform ADAGE analysis.
+
+STRAIN = ['pao1', 'pa14'] # only run pao1 since this is what the eADAGE model is based on
```

Comment:
If you're only running pao1, what is the point of this variable?

Reply:
It's parameterized in case we want to run in on pa14, and then follows the syntax/style of the rest of the snakefiles

Reply:
```suggestion
STRAIN = ['pao1', 'pa14'] 
```

woops good catch! I actually ran it with both, so removing comment